lib/mlir/Dialect/Clift/IR/CliftOps.cpp

```diff
@@ -251,6 +251,21 @@ mlir::LogicalResult clift::ModuleOp::verify() {
 
 //===----------------------------- Statements -----------------------------===//
 
+//===-------------------------------- ForOp -------------------------------===//
+
+mlir::LogicalResult ForOp::verify() {
```

Comment:
We do. It's added in the subsequent commit that also adds the minimal expression ops, because without the expressions there is nothing to test. See [here](https://github.com/revng/revng-c/pull/25/commits/c107f733663ad7a0475c8632fca552af66333639#diff-6b10b0ccf89a76f6de01b3fb1d1c780894a2d8a299beba9ac22cf98471b05383R300).

Reply:
Perfect!